spacepy/pybats/__init__.py

```diff
@@ -422,25 +319,48 @@ def _read_idl_ascii(pbdat, header='units', start_loc=0, keep_case=True):
     in the object's attribute list under 'header'.
 
     Parameters
-    ----------
-    pbdat : PbData object
+    ==========
+    pbdat - PbData object
         The object into which the data will be loaded.
 
     Other Parameters
-    ----------------
-    header : string or **None**
+    ================
+    header - string or **None**
         A string indicating how the header line will be handled; see above.
-    start_loc : int
+    start_loc - int
         Starting location within file of data to read in number of characters.
         Used when reading *.outs files with more than one data frame.
-    keep_case : boolean
+    keep_case - boolean
         If set to True, the case of variable names will be preserved.  If
         set to False, variable names will be set to all lower case.
 
     Returns
-    -------
-    True : Boolean
+    =======
+    True - Boolean
         Returns True on success.
+    
+    Notes
+    =====
+    PyBats assumes little endian byte ordering because
+    this is what most machines use.  However, there is an autodetect feature
+    such that, if PyBats doesn't make sense of the first read (a record length
+    entry, or RecLen), it will proceed using big endian ordering.  If this
+    doesn't work, the error will manifest itself through the "struct" package
+    as an "unpack requires a string of argument length 'X'".
+
+    .. versionchanged:: 0.5.0
+
+        Unstructured data are presented as in the files. When reading
+        3D magnetosphere files, this preserves the 3D block structure,
+        as required for the BATSRUS interpolator in the `Kamodo
+        Heliophysics model readers package
+        <https://github.com/nasa/kamodo>`_. Before 0.5.0, binary
+        unstructured data were sorted in an attempt to put nearby
+        positions close to each other in the data arrays. This sorting
+        was nondeterministic and has been removed; see
+        :meth:`~spacepy.pybats.bats.Bats2d.extract` and
+        :class:`~spacepy.pybats.qotree.QTree` for processing
+        adjacent cells. (ASCII data were never sorted.)
```

Comment:
Why was this moved here? It's specific to IdlFile

Reply:
I just saw a wicked long sectioned docstring in the file, and I added it to the additional user documentation. I can move it back if it makes more sense, but I have it currently under the IdlFile section of the pybats user doc

Reply:
"Notes" is a legit numpydoc section heading, so it can stay in the docstring.

It was in the IdlFile class. As of the fetch I just did (and was the case when I mae this comment) it was in the docstring of `_read_idl_ascii`, which it's completely irrelevant to.

Reply:
I think what happened was I pasted it in the wrong spot on accident, I fixed this now, so it should be resolved